lua/wtf/gpt.lua

```diff
@@ -67,13 +67,29 @@ local function get_api_key()
   return api_key
 end
 
+local function get_base_url()
+  local url = config.options.openai_base_url
+  if url == nil then
```

Comment:
Is the nil check necessary here? As we already have the default API endpoint set in the `default_ops` as a fallback for if one is not specified? Even if you intentionally set it to be `nil` in your config, the default will still be set in this case?